### PR TITLE
NCR embassy + tweaks

### PR DESCRIPTION
**Pull request by SeeminglyTypicalUsername**
## About The Pull Request
Adds an NCR embassy faction (5 slots)
Removes some barbed wire, buffs clinic, armory balance, buffs pipe rifle, removes a set of shutters.
image

## Why It's Good For The Game

People have been complaining about the barbed wire the game needs more tweaking.
More nongang related content is needed.

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Gives the clinic more medical equipment
add: Adds more windows around the map
add: adds in an NCR coat
add: New subtype NCR armor
add: Adds small NCR Embassy
add: Adds more toolbox spawns
add: More metal spawns
add: Gives the clinic a medical bag
add: Gives the clinic a medical vendor
del: Removes the gamer briefcases
add: Adds a heater to the map
del: Removes a wall in the clinic
del: Removes some barbed wire
del: Removes the shutters to the front door of the jungle gym so it is accessible.
tweak: Replaces fully upgraded chem machines with standard models
tweak: Fixes wright capo loadout 
tweak: Edits bank dungeon
tweak: fixes surgery bags
tweak: swaps out ID types
balance: Replaces plasma muskets with wattz in the Salvatore's armory
balance: Buffs pipe rifle to have a 3 round internal magazine.
/:cl:

### CODE REVIEW

**Comment by hermaplusplus** [comment]
Commits [34bb079](https://github.com/StoneAndStar/RockStar/pull/12/commits/34bb079bef4aa2dafbeebeea428ddd27eb62358a), [e36f616](https://github.com/StoneAndStar/RockStar/pull/12/commits/e36f616b5cadc37a9f004f80345517bf3a296037), and [a1f4dcc](https://github.com/StoneAndStar/RockStar/pull/12/commits/a1f4dccc92ac54daf408a863858fbc3ef5e2aeae) are already committed to `maps/newreno` (see #12). Double check your local branch please.

**Comment by hermaplusplus** [comment]
Regarding documentation, your changelog is malformed. See the [pull request template](https://github.com/StoneAndStar/RockStar/blob/main/.github/PULL_REQUEST_TEMPLATE.md?plain=1) for specifics.

**Comment by hermaplusplus** [comment]
Merge conflicts were caused by a rebase-commit issue, which is now resolved. However, the changelog is still malformed - please read through the comments in the [pull request template](https://github.com/StoneAndStar/RockStar/blob/main/.github/PULL_REQUEST_TEMPLATE.md?plain=1); the CL keys that you are using should be altered to match existing keys (e.g. `add`, `del`, `tweak`).

File: _maps/map_files/Reno/Reno-Surface-2.dmm [hunk 1]
```diff
@@ -231,8 +231,8 @@
 /area/f13/building)
 "bX" = (
 /obj/structure/shelf_wood,
-/obj/item/gun/ballistic/rifle/hobo/plasmacaster,
```
Comment (Kitsunemitsu) [inline _maps/map_files/Reno/Reno-Surface-2.dmm:234]: Are plasmacasters this bad?

File: code/__DEFINES/radio.dm [hunk 2]
```diff
@@ -81,6 +81,26 @@
...
+#define RADIO_CHANNEL_MORDINOS "Mordinos"
+#define RADIO_KEY_MORDINOS "m"
+#define RADIO_TOKEN_MORDINOS ":m"
+
+#define RADIO_CHANNEL_WRIGHTS "Wrights"
+#define RADIO_KEY_WRIGHTS "w"
+#define RADIO_TOKEN_WRIGHTS ":w"
+
+#define RADIO_CHANNEL_HOTEL "Hotel"
+#define RADIO_KEY_HOTEL "h"
```
Comment (Kitsunemitsu) [inline code/__DEFINES/radio.dm:101]: Do NOT use :h
That is the default for your department

File: tgui/packages/tgui-panel/styles/goon/chat-light.scss [hunk 3]
```diff
@@ -454,6 +454,26 @@ em {
   color: #f0420d;
 }
 
+.salvatoresradio {
+  color: #FF99CC;
```
Comment (Kitsunemitsu) [inline tgui/packages/tgui-panel/styles/goon/chat-light.scss:458]: I'm not a TGUI Expert, Make sure you THOROUGHLY test this.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Very minor changes and clarifications/

File: code/__DEFINES/radio.dm (hunk 2 above)
Comment (Kitsunemitsu) [reply code/__DEFINES/radio.dm:101]: Just want to clarify, if it works, it works.
There may be serious unintended side effects tho

File: _maps/map_files/Reno/Reno-Surface-2.dmm (hunk 1 above)
Comment (SeeminglyTypicalUsername) [reply _maps/map_files/Reno/Reno-Surface-2.dmm:234]: I don't think it's a balanced weapon to start with. If they had to buy it, it would be fine.

**Review by Kitsunemitsu — APPROVED** [review]

**Pull request merged**